src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { useAppContext } from '../context/AppContext';
+import { User } from '../User';
+
+export const PeopleList = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isErrorToGetPeople, setIsErrorToGetPeople] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const { hasClickedPeopleLink } = useAppContext();
+  const { slug } = useParams();
+  const selectId = slug || '';
+  const selectedUser = people.find(p => p.slug === selectId);
+
+  useEffect(() => {
+    if (hasClickedPeopleLink) {
+      getPeople()
+        .then((allPersons) => setPeople(allPersons))
+        .catch(() => {
+          setIsErrorToGetPeople(true);
+          throw new Error(' Something went wrong');
+        })
+        .finally(() => setIsLoading(false));
+    }
+  }, [hasClickedPeopleLink]);
+
+  return (
+    <div className="block">
+      <div className="box table-container">
+        <h1 className="title">People Page</h1>
```

Comment:
must be outside the "block"
![image](https://github.com/mate-academy/react_people-table-basics/assets/110335429/728d51ce-da7c-4f4b-addc-78c5ed9a90d7)